static/js/zamboni/devhub.js

```diff
@@ -376,6 +380,16 @@ function addonFormSubmit() {
     })(parent_div);
 }
 
+function refreshAddonBasicInformation($form) {
+    // Retrieve the div containing the updated name and then its value
+    update_name_div = $form.find('[data-name=name]')
+    updated_name = update_name_div.text()
+    // Get the element to update in the top-left quadrant and update it with the new name
+    // The element being defined in devhub/addons/edit.html file belongs to the class addon-name
+    addon_name_tag = $('.addon-name')
```

Comment:
`$('.addon-name').text(updated_name)` is enough

Reply:
@diox simplified the code.